react/AutocompleteBlock.tsx

```diff
@@ -341,7 +342,7 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
                             return (
                               <span
                                 key={item.itemId}
-                                className={`mr4 ${handles.skuSelection}`}
+                                className={`mr4 ${handles.skuSelection} ${item.itemId === selectedItem.value && handles.skuSelectionSelected}`}
```

Comment:
The goal here is to just add a `true` or `false` to the className?

Reply:
Yes! Only to get when is selected